timemachine/fe/single_topology.py

```diff
@@ -178,23 +196,31 @@ def setup_dummy_interactions(
         if all([a in dga for a in idxs]):
             dummy_improper_idxs.append(tuple([int(x) for x in idxs]))
             dummy_improper_params.append(params)
+
+    # certain configuration of chiral states are symmetrizable
+    #
+    #        all allowed geometries              |  disallowed geometry
+    #     center is core     center is not core  |
+    #    d      c      c      d      d      d    |    c      c
+    #    .      |      |      .      .      .    |    .      |
+    #    c      c      c      d      d      d    |    d      c
+    #   . .    . .    / .    . .    . .    . .   |   . .    / \
+    #  d   d  d   d  c   d  c   d  c   c  d   d  |  c   c  c   c
+    dgc = dummy_group + list(core_atoms)
     for idxs, params in zip(chiral_atom_idxs, chiral_atom_params):
-        # consider a few cases: (1) all idxs dummy, (2) a single root_anchor_atom and 3 dummies, (3) other
         center, i, j, k = idxs
-        case_1 = all(a in dummy_group for a in (center, i, j, k))
-        three_in_dummy_group = sum(a in dummy_group for a in (center, i, j, k)) == 3
-        one_is_root_anchor_atom = sum(a == root_anchor_atom for a in (center, i, j, k)) == 1
-        case_2 = three_in_dummy_group and one_is_root_anchor_atom
-        # TODO: maybe add more cases, if we convince ourselves they are also factorizable...
-        #    e.g. if center is root_anchor_atom, and at most one of (i, j, k) is neighbor_anchor_atom
-        #   (and maybe also if root_anchor_atom, neighbor_anchor_atom are swapped in above statement)
-        if case_1 or case_2:
-            dummy_chiral_atom_idxs.append(tuple(int(x) for x in idxs))
-            dummy_chiral_atom_params.append(params)
+        if all([a in dgc for a in (center, i, j, k)]):
```

Comment:
nit: Avoid reconstructing tuple
 
```suggestion
        if all([a in dgc for a in idxs]):
```

Reply:
willfix